samples/network/README.md

```diff
@@ -88,3 +88,7 @@ Each file is a separate code sample that no dependency on other files. You can l
 ## Resources
 
 - https://github.com/Azure/azure-sdk-for-python
+
+## Additional doc
+
+- Load balancer : https://github.com/Azure-Samples/azure-samples-python-management/tree/master/doc/network-manage-loadbalancer
```

Comment:
Load balancer -> Getting started with public and internal load balancers in Python

Also, use a relative link?